common/server_protocolhandler.cpp

```diff
@@ -373,6 +383,20 @@ void Server_ProtocolHandler::pingClockTimeout()
 
     if (timeRunning - lastDataReceived > server->getMaxPlayerInactivityTime())
         prepareDestroy();
+
+    if (QString::fromStdString(userInfo->privlevel()).toLower() == "none")
+        if ((timeRunning - lastActionReceived) == round(server->getIdleClientTimout() *.9)) {
```

Comment:
Originally the runction used `ceil` rather than `round` and I prob be switching things back to that.  If we are introducing another counter would it make more since to add a bool into the equation and turn the bool on/off if the warning has been sent and have the bool clear at some point rather than making a best guess in the event the ping clock timer is set to some number that the math doesnt really work for?